docs/changelog/111285.yaml

```
@@ -0,0 +1,5 @@
+pr: 111285
+summary: "[Bugfix] Add `accessDeclaredMembers` permission for `ObjectMapper` to work"
```

Nitpick, just to make the fix more clear. 

```suggestion
summary: "[Bugfix] Add `accessDeclaredMembers` permission to fix bug in rendered search application templates"
```